packages/integration-sdk-core/src/types/storage.ts

```diff
@@ -30,8 +30,13 @@ export interface GraphObjectStore {
     onRelationshipsFlushed?: (relationships: Relationship[]) => Promise<void>,
   ): Promise<void>;
 
+  /**
+   * @deprecated Use findEntity
+   */
   getEntity({ _key, _type }: GraphObjectLookupKey): Promise<Entity>;
 
+  findEntity(_key: string): Promise<Entity | undefined>;
```

Comment:
Officially added `findEntity` to the `GraphObjectStore` spec, and deprecated `getEntity`.